src/repos/UserRepo.ts

```diff
@@ -38,6 +53,9 @@ const createUser = async (
  */
 const followPublication = async (uuid: string, pubID: string): Promise<User> => {
   const user = await UserModel.findOne({ uuid });
+  if (!user) {
```

Comment:
can we swap this with checking
```
if (user) {
   user.followedPublications.push(Object.assign(new PublicationID(), { id: pubID }));
   return user.save();
}
return user;
```

so that we can keep consistent with the other ifs in the codebase. Do this for lines 68-70 as well.